infra/debian/compiler/one-compiler.install

```diff
@@ -51,6 +51,8 @@ usr/bin/visqlib/Palette.py usr/share/one/bin/visqlib/
 usr/bin/visqlib/QErrorComputer.py usr/share/one/bin/visqlib/
 usr/bin/visqlib/Util.py usr/share/one/bin/visqlib/
 usr/bin/visqlib/DotBuilder.py usr/share/one/bin/visqlib/
+usr/bin/circle/* usr/share/one/bin/circle/
```

Comment:
circle python interface is needed for visq

Reply:
what are the files in our source repo for these?

Reply:
is `*` appropriate usage? why not use each file name? any files introduce to `use/bin/circle` may be installed without any control.

Reply:
Python interface for circle model (generated by `pics` module) exists under `usr/bin/circle`. I used `*` as there are too many (156) files. 

FYI, the size of all files is 704 KB. For bionic, our current package size is 3 MB, and our venv size is 1.7 GB. This PR will increase our package size a bit, but I think that's negligible considering the total framework size.

<details>

```
tree build/release.install/bin/circle
build/release.install/bin/circle
├── AbsOptions.py
├── ActivationFunctionType.py
├── AddNOptions.py
├── AddOptions.py
├── ArgMaxOptions.py
├── ArgMinOptions.py
├── AssignVariableOptions.py
├── ATan2Options.py
├── BatchMatMulOptions.py
├── BatchToSpaceNDOptions.py
├── BCQFullyConnectedOptions.py
├── BCQGatherOptions.py
├── BidirectionalSequenceLSTMOptions.py
├── BidirectionalSequenceRNNOptions.py
├── BroadcastToOptions.py
├── BucketizeOptions.py
├── Buffer.py
├── BuiltinOperator.py
├── BuiltinOptions.py
├── CallOnceOptions.py
├── CallOptions.py
├── CastOptions.py
├── CombinerType.py
├── ConcatEmbeddingsOptions.py
├── ConcatenationOptions.py
├── Conv2DOptions.py
├── Conv3DOptions.py
├── CosOptions.py
├── CumsumOptions.py
├── CustomOptionsFormat.py
├── CustomQuantization.py
├── DataFormat.py
├── DensifyOptions.py
├── DepthToSpaceOptions.py
├── DepthwiseConv2DOptions.py
├── DequantizeOptions.py
├── DimensionMetadata.py
├── DimensionType.py
├── DivOptions.py
├── DynamicUpdateSliceOptions.py
├── EmbeddingLookupSparseOptions.py
├── EqualOptions.py
├── ExpandDimsOptions.py
├── ExpOptions.py
├── FakeQuantOptions.py
├── FillOptions.py
├── FloorDivOptions.py
├── FloorModOptions.py
├── FullyConnectedOptions.py
├── FullyConnectedOptionsWeightsFormat.py
├── GatherNdOptions.py
├── GatherOptions.py
├── GeluOptions.py
├── GreaterEqualOptions.py
├── GreaterOptions.py
├── HardSwishOptions.py
├── HashtableFindOptions.py
├── HashtableImportOptions.py
├── HashtableOptions.py
├── HashtableSizeOptions.py
├── IfOptions.py
├── __init__.py
├── InstanceNormOptions.py
├── Int32Vector.py
├── L2NormOptions.py
├── LeakyReluOptions.py
├── LessEqualOptions.py
├── LessOptions.py
├── LocalResponseNormalizationOptions.py
├── LogicalAndOptions.py
├── LogicalNotOptions.py
├── LogicalOrOptions.py
├── LogSoftmaxOptions.py
├── LSHProjectionOptions.py
├── LSHProjectionType.py
├── LSTMKernelType.py
├── LSTMOptions.py
├── MatrixDiagOptions.py
├── MatrixSetDiagOptions.py
├── MaximumMinimumOptions.py
├── Metadata.py
├── MirrorPadMode.py
├── MirrorPadOptions.py
├── Model.py
├── MulOptions.py
├── NegOptions.py
├── NonMaxSuppressionV4Options.py
├── NonMaxSuppressionV5Options.py
├── NotEqualOptions.py
├── OneHotOptions.py
├── OperatorCode.py
├── Operator.py
├── PackOptions.py
├── Padding.py
├── PadOptions.py
├── PadV2Options.py
├── Pool2DOptions.py
├── PowOptions.py
├── QuantizationDetails.py
├── QuantizationParameters.py
├── QuantizeOptions.py
├── RandomOptions.py
├── RangeOptions.py
├── RankOptions.py
├── ReadVariableOptions.py
├── ReducerOptions.py
├── ReshapeOptions.py
├── ResizeBilinearOptions.py
├── ResizeNearestNeighborOptions.py
├── ReverseSequenceOptions.py
├── ReverseV2Options.py
├── Rfft2dOptions.py
├── RNNOptions.py
├── ScatterNdOptions.py
├── SegmentSumOptions.py
├── SelectOptions.py
├── SelectV2Options.py
├── SequenceRNNOptions.py
├── ShapeOptions.py
├── SignatureDef.py
├── SkipGramOptions.py
├── SliceOptions.py
├── SoftmaxOptions.py
├── SpaceToBatchNDOptions.py
├── SpaceToDepthOptions.py
├── SparseIndexVector.py
├── SparseToDenseOptions.py
├── SparsityParameters.py
├── SplitOptions.py
├── SplitVOptions.py
├── SquaredDifferenceOptions.py
├── SquareOptions.py
├── SqueezeOptions.py
├── StridedSliceOptions.py
├── SubGraph.py
├── SubOptions.py
├── SVDFOptions.py
├── TensorMap.py
├── Tensor.py
├── TensorType.py
├── TileOptions.py
├── TopKV2Options.py
├── TransposeConvOptions.py
├── TransposeOptions.py
├── Uint16Vector.py
├── Uint8Vector.py
├── UnidirectionalSequenceLSTMOptions.py
├── UniqueOptions.py
├── UnpackOptions.py
├── UnsortedSegmentMaxOptions.py
├── UnsortedSegmentProdOptions.py
├── UnsortedSegmentSumOptions.py
├── VarHandleOptions.py
├── WhereOptions.py
├── WhileOptions.py
└── ZerosLikeOptions.py
```
</details>

> any files introduce to use/bin/circle may be installed without any control.

It would be better to copy only python files (*.py). I'll update the PR after test.

Reply:
Ah, I wasn't concerned about the size.
I don't know much about Debian package generation and don't know using `*` is usual thing or bad practice.

Reply:
https://manpages.debian.org/testing/debhelper/dh_install.1.en.html

> debian/package.install
> ...
>  You may use wildcards in the names of the files to install. 

it's OK :)